apps/myaccount/src/components/multi-factor-authentication/authenticators/smsotp-authenticator.tsx

```diff
@@ -22,14 +22,16 @@ import { isEmpty } from "lodash";
 import React, { useEffect, useState } from "react";
 import { useTranslation } from "react-i18next";
 import { useDispatch, useSelector } from "react-redux";
-import { Form, Grid, Icon, List } from "semantic-ui-react";
+import {Form, Grid, Icon, List, Placeholder} from "semantic-ui-react";
```

Comment:
```suggestion
import { Form, Grid, Icon, List, Placeholder } from "semantic-ui-react";
```

Reply:
Addressed in https://github.com/wso2/identity-apps/pull/1275/commits/375a1a7a8b040739aaae46cb9b9657e461a0bf70